aas_repository_semantic_matcher/matcher.py

```diff
@@ -135,12 +135,14 @@ def matches_to_xml(self) -> str:
                 comment=f"Created by indexing {self.repository_endpoint} via https://github.com/rwth-iat/aas-repository-semantic-matcher"  # Todo
             )
             for index_element in index_elements:
+                # The index_key is the semantic ID, which is used as element class in this context.
                 xmlization.add_element(
                     link=link,
                     element_id=self._create_element_id(index_element.semantically_identified_referable),
                     name=f"{index_element.semantically_identified_referable.__class__.__name__}",
                     model=self._get_identifiable_identifier(index_element.semantically_identified_referable),
-                    tool="BaSyx"
+                    tool="BaSyx",
+                    element_class=str(index_key)
```

Comment:
```suggestion
                    element_class=str(index_key)  # The index_key is the semantic ID, which is used as element class in this context.
```

Can you move the comment from line 138 down to this line (or above this line, if it gets too long)?
It makes more sense to note that where you need it, I was a bit confused as why the commenz was mentioning this above.